generators/app/templates/settings.json

```diff
@@ -1,5 +0,0 @@
-{
-    "files.associations": {
-        "dockerfile.*": "dockerfile"
```

Comment:
The right-hand side requires lowercase. The left-hand side has been insensitive in my testing so I made it lowercase for consistency.